includes/core/classes/class-setup.php

```diff
@@ -135,6 +136,36 @@ public function block_category( $block_categories ) {
 		return $block_categories;
 	}
 
+	/**
+	 * Display custom admin notice function
+	 *
+	 * @return void
+	 */
+	public function timezone_check_admin_notice() {
+		$timezone = get_option( 'timezone_string' );
+		if ( $timezone ) {
+			return;
+		}
+		$class = 'notice notice-error is-dismissible';
+		printf(
+			wp_kses(
+				// translators: %1$s classes for admin_notice, %2$s url link to setting.
+				__( '<div class="%1$s"><p>Please set <a href="%2$s">your timezone</a> in order to ensure proper GatherPress settings!</p></div>', 'gatherpress' ),
```

Comment:
don't need the `div` or `p` in there, sorry if that wasn't clear. just need the `a` since that's in with the translated string.